python_ta/reporters/templates/stylesheet.css

```diff
@@ -15,6 +15,30 @@ header {
     text-align: center;
 }
 
+.main-grid {
+    display: grid;
+    grid-template-columns: 20em 1fr
+}
+
+aside {
+    padding-left: 16px;
+    padding-right: 16px;
+}
+
+#sidebar {
+    height: 100vh;
+    width: 300px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+
+    top: 0;
+    position:sticky;
+
+    overflow-y: auto;
+    overscroll-behavior-y: contain;
```

Comment:
I don't think this is necessary, nor the `position: sticky`. See comments about using css grid below.